### PR TITLE
Download Initial Data

### PR DESCRIPTION
**Pull request by gwaybio**
I shared this script on slack previously, but I think it would work well as the tipping off point for this repository. 

Note that the script doesn't actually work right now :roll_eyes: - there is something buggy about GEO ftp. Hopefully this gets squared away soon. We can also wait to merge this until it works if people feel strongly.

I also added `data/` to the `.gitignore`. My thoughts on what to do with the data are in #1  - we can discuss there alternative options too.

### CODE REVIEW

**Comment by gwaybio** [comment]
We should also discuss how we should handle PR review too @huqiwen0313 @jaclyn-taroni

File: download_data.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# czi-cancer-data
+# 2018 Gregory Way
+
+# Script to download gene expression and covariate data from 4 single cell gene
+# expression analyses. The script will fetch, decompress, and rename each data set
+# listed below
+
+mkdir --parents data/raw
```
Comment (jaclyn-taroni) [inline download_data.sh:10]: Should this be followed by `cd data && cd raw`?

**Review by jaclyn-taroni — APPROVED** [review]
One comment/question about directories in the data download shell script, otherwise LGTM 👍

File: download_data.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
```
Comment (huqiwen0313) [inline download_data.sh:2]: Is it possible to add some descriptions for these samples or have a file described what these samples are.

**Review by huqiwen0313 — APPROVED** [review]

File: download_data.sh (hunk 1 above)
Comment (gwaybio) [reply download_data.sh:10]: i process files in this script too - it moves the data to appropriate directory

**Comment by gwaybio** [comment]
> Note that the script doesn't actually work right now :roll_eyes: - there is something buggy about GEO ftp. 

Can confirm that it now works!

File: download_data.sh (hunk 1 above)
Comment (jaclyn-taroni) [reply download_data.sh:10]: Oh yea, my bad

File: download_data.sh (hunk 2 above)
Comment (gwaybio) [reply download_data.sh:2]: yeah, I think that is a good idea. I will add a brief description here, but a more thorough description should exist as some sort of table, text, or FAQ in the README or github wiki

**Comment by gwaybio** [comment]
Thanks @huqiwen0313 - I added a brief description to the data in the current download file. Also note that this script should have a couple datasets also added to it (in #3 and #4)

**Pull request merged**